test/tools/integration/provision_master.sh

```diff
@@ -42,17 +42,17 @@ if ! which kubelet; then
 EOF
   apt-get update
   apt-get install -y kubelet kubeadm kubectl
-  kubeadm init --apiserver-advertise-address=$ADDR --pod-network-cidr=10.244.0.0/16
+  kubeadm init --apiserver-advertise-address=$ADDR --pod-network-cidr=192.168.0.0/16
 fi
 if ! ls \$HOME/.kube/config; then
   mkdir -p \$HOME/.kube
   cp -i /etc/kubernetes/admin.conf \$HOME/.kube/config
   kubectl taint nodes --all node-role.kubernetes.io/master-
 fi
-if ! ls kube-flannel.yml; then
-  curl -LO https://raw.githubusercontent.com/coreos/flannel/v0.9.1/Documentation/kube-flannel.yml
-  kubectl apply -f kube-flannel.yml
-fi
+
```

Comment:
Please leave this script declaratively and only let it apply stuff that's not already there, makes local testing much faster

Reply:
Also, are you sure flannel does not work? I would prefer to avoid having e2e tests here run with a different networking solution